docs/versioned_docs/version-v8.5.x/03-light-clients/02-localhost/03-client-state.md

```diff
@@ -33,7 +33,7 @@ It is possible to disable the localhost client by removing the `09-localhost` en
 
 The latest height is updated periodically through the ABCI [`BeginBlock`](https://docs.cosmos.network/v0.47/building-modules/beginblock-endblock) interface of the 02-client submodule in core IBC.
 
-[See `BeginBlocker` in abci.go.](https://github.com/cosmos/ibc-go/blob/09-localhost/modules/core/02-client/abci.go#L12)
+[See `BeginBlocker` in abci.go.](https://github.com/cosmos/ibc-go/blob/main/modules/core/02-client/abci.go#L28)
```

Comment:
Given that this is in the versioned docs, I would prefer to change the link to use the code from the same branch: https://github.com/cosmos/ibc-go/tree/release/v8.5.x

Please also take a look and make sure the link is correct in the other versioned docs.

Reply:
I followed your advice and applied the changes to the versioned docs. 

The link https://github.com/cosmos/ibc-go/blob/3077be12f11e9859609028d2b27971a8c6a8fab1/modules/core/02-client/abci.go#L12 is right. I missed.

As a result, I updated the link path and modified pr.

Additionally, I reviewed other versions of the versioned documents. The path for version 7.8.x is correct, and no documentation exists for versions other than 8.5.x.

Therefore, I concluded that only the 8.5.x version needed fixing.

Thank you for your advice. If you notice any further issues or mistakes in my work, please let me know.